From e2c89aaa6b99586c72d55024e4c42c0a331864f6 Mon Sep 17 00:00:00 2001
From: He Sheng <hesheng@wxiat.com>
Date: Fri, 6 May 2022 09:30:28 +0800
Subject: [PATCH] sw64: uapi: include generic param.h
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: e2c89aaa6b99586c72d55024e4c42c0a331864f6
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

Most macro definitions are the same with generic version.

Signed-off-by: He Sheng <hesheng@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/include/uapi/asm/param.h | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/arch/sw_64/include/uapi/asm/param.h b/arch/sw_64/include/uapi/asm/param.h
index 16c4934c937e..d38e8202dd97 100644
--- a/arch/sw_64/include/uapi/asm/param.h
+++ b/arch/sw_64/include/uapi/asm/param.h
@@ -2,15 +2,8 @@
 #ifndef _UAPI_ASM_SW64_PARAM_H
 #define _UAPI_ASM_SW64_PARAM_H
 
-#define HZ		100
-
 #define EXEC_PAGESIZE	8192
 
-#ifndef NOGROUP
-#define NOGROUP		(-1)
-#endif
-
-#define MAXHOSTNAMELEN	64	/* max length of hostname */
-
+#include <asm-generic/param.h>
 
 #endif /* _UAPI_ASM_SW64_PARAM_H */
-- 
2.33.0

